pkg/server/plugin/datastore/sqlite/sqlite.go

```diff
@@ -124,8 +125,21 @@ func (ds *sqlitePlugin) AppendBundle(req *datastore.Bundle) (*datastore.Bundle,
 	}
 	model.CACerts = caCerts
 
-	// Set the new values
-	model.CACerts = append(model.CACerts, newModel.CACerts...)
+	// Make sure we don't already have the certs stored
+	for _, newCA := range newModel.CACerts {
+		var found bool
+		for _, c := range model.CACerts {
```

Comment:
I wonder if this for loop (and the if !found) could be refactored to look like:
```
if !model.Contains(newCA) {
   model.Append(newCA)
}
```
I'm asking because maybe there are some design considerations that I'm not taking into account.

Reply:
I looked for `Contains` and `Append` methods that work like this but couldn't find any... It might be possible to define them directly on the model though, here: https://github.com/spiffe/spire/blob/master/pkg/server/plugin/datastore/sqlite/models.go#L18-L23

Is that what you meant?

Reply:
Yeah, that's what I meant. Only if you think it worth it. I there is not going to be any other use case for Contains or Append, then maybe it doesn't worth it.

Reply:
I think this might be the only place we would use those, but your suggestion is definitely more readable - fixed!